.github/workflows/learning-web-branch.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
         working-directory: learning/web
         run: yarn build
         env:
-          PATH_PREFIX: ""
+          PATH_PREFIX: "/"
```

Comment:
does this change make a difference?

Reply:
Nope - it just makes it explicit where it might have been ambiguous before, and makes it consistent with the other workflow files.